src/app/examples/store/index.ts

```diff
@@ -1,17 +1,26 @@
 import { createFeatureSelector, createSelector } from '@ngrx/store';
+import * as fromHackerNews from './hacker-news/hacker-news.reducer';
 import * as fromExampleData from './example-data/example-data.reducer';
 import * as fromRoot from '../../core/store';
 
 export interface ExamplesState {
     exampleData: fromExampleData.State;
 }
 
+export interface HackerNewsState {
+    articleData: fromHackerNews.State;
```

Comment:
This is pretty close, but you probably don't actually need to create a whole additional feature state, but just add your state slice to the ExamplesState, like:

```
export interface ExamplesState {
     exampleData: fromExampleData.State;
     articleData: fromHackerNews.State;
 }

export const reducers = {
     exampleData: fromExampleData.reducer
     articleData: fromHackerNews.reducer
 };
```